workspaces/redhat-resource-optimization/docs/adrs/0001-optimizations-plugin-for-red-hat-developer-hub-phase-1.md

```diff
@@ -0,0 +1,50 @@
+# ADR-0001: Optimizations plugin for Red Hat Developer Hub - Phase 1
+
+## Status
+
+Accepted
+
+## Context
+
+[Resource Optimization](https://docs.redhat.com/en/documentation/cost_management_service/1-latest/html-single/getting_started_with_resource_optimization_for_openshift/index#what_is_resource_optimization_for_openshift) is a feature that extends the Cost Management service with the goal of helping users to rightsize workloads running on their OpenShift clusters.  
+Currently, this feature is exposed exclusively through [OCM](https://console.redhat.com/openshift) but, with the introduction of Red Hat Developer Hub ([Backstage](https://backstage.io)) we can bring the same feature to the platform via a plugin.
```

Comment:
@jkilzi should it be OCM or Red Hat Insights for Openshift?

Reply:
As it is today, the location heirarchy of this service is: c.rh.c/Openshift/Cost management/Optimizations. Since c.rh.c/Openshift is actually what we call OCM, I don't think we should change it. I beleive what becomes confusing is the fact that Optimizations for RHEL can be found under Red Hat Insights 🤷  

![image](https://github.com/user-attachments/assets/865d4cc1-6569-462f-a618-52c90fe2f716)